documentation/peer-testing-1/peer-testing-report.md

```diff
@@ -27,18 +27,37 @@ In total, \_\_\_ individuals took part in our peer testing sessions. The members
 
 #### High
 
--
+- Timezone field does not "glow" red if the user did not enter anything, it only shows the text "please select your time-zone". (mentor and mentee form)
+- When signing in, if the user does not enter anything for email and password, it just says "please input something" for both fields, and then a pop-up that says "the email address is badly formatted" however both fields are blank. (login page)
+- From the above point, we should change "please input something" to "please input password" or "please input email", for the red text that pops up at the bottom of the field if the user does not enter an email address in the login page. (login page)
+- If the user enters a incorrect username and password, the website creates a popup that is "wordy" and maybe difficult to understand. (login page)
+- We should make different messages depending on of if the user did not enter any information in login screen (password/email) or if the user entered information but the information is wrong/does not have an account. (login page)
+- From above points, change the login pop up messages depending on what is and is not filled in. Change the red text that popups at the bottom of the field depending on what is and is not filled in. (login page)
+- Make the "organization" field not mandatory. This way, people without an organziation can sign up.
```

Comment:
I would remove this. From speaking with the client, it sounds like she only wants individuals from Organizations to be Mentors.

Reply:
Sounds good! removed!